app/upgrades/v2/upgrades.go

```diff
@@ -12,8 +15,15 @@ func CreateUpgradeHandler(
 	configurator module.Configurator,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
```

Comment:
in SDK we use `from` or `fromVM` to make it clear that this map contains the existing versions  (pre-upgrade)